e2e/tests/LoggedIn/publish.e2e.spec.ts

```diff
@@ -446,14 +443,13 @@ test.describe('Publication flow', () => {
 
         // Preview and check preview draft publication
         await page.locator(PageModel.publish.previewButton).click();
-        await checkPublication(page, hypothesisPublication);
+        await checkPublication(page, hypothesisPublication, [Helpers.user1]);
 
         // Publish and check live publication
         await page.locator(PageModel.publish.draftEditButton).click();
-        await page.waitForResponse((response) => response.url().includes('/references') && response.ok());
```

Comment:
These "waitForResponse" calls seemed to be keeping the tests more stable - from trying the tests out on this branch they are being flakier than before. Can we keep them?

Reply:
The api call for /references has been moved from CSR to SSR. That means there's no http call to /references from the client anymore, which makes all tests to fail waiting for response.
<img width="1178" alt="image" src="https://github.com/JiscSD/octopus/assets/48569671/3d272e98-eaf4-48cb-bc74-60703e3b1e33">

Reply:
Ah I see, that's ok then